service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -213,6 +214,11 @@ private DocmosisHearingBooking buildHearingBooking(HearingBooking hearing) {
     }
 
     private String buildHearingVenue(HearingBooking hearing) {
+        if (hearing.getPreviousHearingVenue() != null
+            && YES.getValue().equals(hearing.getPreviousHearingVenue().getUsePreviousVenue())) {
+            String venueAddress = hearing.getPreviousHearingVenue().getPreviousVenue();
+            return (hearing.isRemote()) ? String.format(REMOTE_HEARING_VENUE, venueAddress) : venueAddress;
```

Comment:
The bracket is not needed for `(hearing.isRemote())`

Reply:
and can just use `format()` instead of `String.format` since static method was imported.
`import static java.lang.String.format;`